src/calculator/calc.oxide

```diff
@@ -0,0 +1,12 @@
+{
+  "displayName": "Calculator",
+  "description": "Remarkble calculator",
+  "bin": "/home/root/calc",
+  "icon": "/opt/etc/draft/icons/calculator.png",
+  "type": "foreground",
+  "events": {
+    "pause": "/opt/koreader/fbdepth -d 16 -r 1",
+    "stop": "/opt/koreader/fbdepth -d 16 -r 1",
+    "resume": "/opt/koreader/fbdepth -d 8 -r 1"
+  }
+}
```

Comment:
this file is unnecessary, can remove. all launchers support draft format (or have a way of importing draft files if not natively supported)